src/backend/controllers/battleController.js

```diff
@@ -181,10 +182,29 @@ exports.reportResults = async (req: Request, res: Response) => {
 					.status(404)
 					.json({ msg: 'Could not find userTwo'});
 			}
+	
+			const newUserOneElo = eloRating.ifTies(userOne.stats.elo, userTwo.stats.elo);
+			const newUserTwoElo = eloRating.ifTies(userTwo.stats.elo, userOne.stats.elo);
+			
+			// We want eloExchanged to be a positive value so we take whoever's increased and subtract 
```

Comment:
These 8 lines can be just `battle.eloExchanged = Math.abs(userOne.stats.elo-newUserOneElo);`